### ISSUE TITLE
Can't pan canvas by scrolling with trackpad

### ISSUE THREAD

**Issue by iwehrman** [adobe-photoshop/spaces-design#1941]
Windows only. It actually does scroll very slowly sometimes, but only vertically.

**Comment by iwehrman** [comment]
Pinch to zoom in/out also does not work.

**Comment by iwehrman** [comment]
It's possible this is a VM artifact. Would either @chadrolfs or @ktaki mind verifying on a real live PC?

**Comment by ktaki** [comment]
Right. Panning with trackpad failed on my newly set up Surface Pro.

**Comment by chadrolfs** [comment]
Pan/zoom touch gestures on surface work tho (2-finger pinch)

**Comment by chadrolfs** [comment]
https://watsonexp.corp.adobe.com//#bug=4037333

**Comment by iwehrman** [comment]
When I pan with the trackpad, the canvas just seems to go down and off the screen no matter what. Windows pgdev.715.

**Comment by chadrolfs** [comment]
Seeing same thing on playground.1620

**Comment by chadrolfs** [comment]
Might defer to M4.  Jesper logging bug with MS today.

**Comment by chadrolfs** [comment]
Watson deferred, moving to M4
